src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/ProceedOfflineCallbackHandler.java

```diff
@@ -24,13 +25,15 @@
 public class ProceedOfflineCallbackHandler extends CallbackHandler {
 
     private static final List<CaseEvent> EVENTS = Collections.singletonList(PROCEEDS_IN_HERITAGE_SYSTEM);
-    private static final String TASK_ID = "ProceedOffline";
+    private static final List<String> TASK_IDS =
+        Arrays.asList("ProceedOffline", "ProceedOfflineForResponseToDefence", "ProceedOfflineForUnregisteredFirm",
+                      "ProceedOfflineForUnRepresentedSolicitor", "ProceedOfflineForNonDefenceResponse");
 
     private final ObjectMapper objectMapper;
 
     @Override
-    public String camundaActivityId(CallbackParams callbackParams) {
-        return TASK_ID;
+    public List<String> camundaActivityIds(CallbackParams callbackParams) {
```

Comment:
Not sure, why we are returning multiple task ids instead of one taskid

Reply:
There are multiple calls to the same event from different places across the camunda processes, and some of them are different ids. I would say they should probably the same id but then you'd have to go down that rabbit hole and investigate. For now we just want to block sequence of events...